torch/csrc/jit/tensorexpr/ir.h

```diff
@@ -787,6 +795,16 @@ class Intrinsics : public CallNode<Intrinsics> {
         return "fmod";
       case kRemainder:
         return "remainder";
+      case kLgamma:
+        return "lgamma";
+      case kExpm1:
+        return "expm1";
+      case kErfc:
+        return "erfc";
+      case kFrac:
+        return "frac";
+      case kReciprocal:
```

Comment:
Should we have this, or just use division to represent it?

Reply:
I'll use division. I'll update this PR